src/Controller/FrameworksController.php

```diff
@@ -30,6 +30,13 @@ class FrameworksController extends AbstractController
      */
     protected $api;
 
+    /**
+     * Default filter for agreement statuses
+     *
+     * @var string
+     */
+    protected const DEFAULTSTATUS = 'all';
```

Comment:
This seems an odd requirement from the API. The status field can be optional and it returns all results by default. 
That would be a better design, if we think about external API users.